content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -163,8 +163,17 @@ Events: <none>
 ## Get the Certificate Signing Request Approved
 
 Approving the certificate signing request is either done by an automated
```

Comment:
> Approving the [certificate signing request](/docs/reference/access-authn-authz/certificate-signing-requests/) is either done by an automated approval process or on a one off basis by a cluster administrator. 

I believe this would benefit from referencing existing docs.